interface/patient_file/report/custom_report.php

```diff
@@ -55,8 +55,9 @@
                        $GLOBALS['pdf_language'],
                        true, // default unicode setting is true
                        'UTF-8', // default encoding setting is UTF-8
-                       array($GLOBALS['pdf_left_margin'],$GLOBALS['pdf_top_margin'],$GLOBALS['pdf_right_margin'],$GLOBALS['pdf_bottom_margin'])
-          ); 
+                       array($GLOBALS['pdf_left_margin'],$GLOBALS['pdf_top_margin'],$GLOBALS['pdf_right_margin'],$GLOBALS['pdf_bottom_margin']),
+                       $_SESSION['language_direction'] == 'rtl' ? true : false
+                   );
```

Comment:
Wouldn't it make more sense to set the default font here via $pdf->setDefaultFont() rather than hard-code it within htmlpdf?